src/include/daos_srv/dtx_srv.h

```diff
@@ -64,7 +67,13 @@ struct dtx_handle {
 					 /* Modified shared items: object/key */
 					 dth_modify_shared:1,
 					 /* The DTX entry is in active table. */
-					 dth_active:1;
+					 dth_active:1,
+					 /* Leader oid is touched. */
+					 dth_touched_leader_oid:1,
```

Comment:
Minor: This flag indicates leader oid is already inserted in the oid array? If so, "dth_leader_oid_inserted" could be better?

Reply:
Hm, if only single EC object is modified by the DTX, then there will be no EC objects ID array, instead, such OID is stored in dtx_handle::dth_leader_oid. If we name it as "dth_leader_oid_inserted", it is some misguided that we would always allocated the EC objects ID array.